### PR TITLE
Update supported versions list for MDE for Linux

### PR DESCRIPTION
**Pull request by ejazr83**
The supported versions of Redhat Linux does not mention RHEL 8.x However, other documentation does mention RHEL 8 https://docs.microsoft.com/en-us/microsoft-365/security/defender-endpoint/linux-install-manually?view=o365-worldwide#client-configuration

Can we add a line to mention RHEL 8.x or higher?

I have proposed adding the below line:
- Red Hat Enterprise Linux 8.x

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks

File: microsoft-365/security/defender-endpoint/microsoft-defender-endpoint-linux.md [hunk 1]
```diff
@@ -68,6 +68,7 @@ If you experience any installation failures, refer to [Troubleshooting installat
 
 - Supported Linux server distributions and x64 (AMD64/EM64T) versions:
 
+  - Red Hat Enterprise Linux 8.x
   - Red Hat Enterprise Linux 6.7 or higher
   - Red Hat Enterprise Linux 7.2 or higher
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/microsoft-defender-endpoint-linux.md]:
```suggestion
  - Red Hat Enterprise Linux 6.7 or higher
  - Red Hat Enterprise Linux 7.2 or higher
  - Red Hat Enterprise Linux 8.x
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @ejazr83, kindly commit the minor suggested change (cosmetic).

**Comment by ejazr83** [comment]
@JohanFreelancer9 Thanks for reviewing, I have made the change.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@Dansimp Please merge this PR. Thanks

**Comment by ejazr83** [comment]
@Dansimp  and @yogkumgit , is there anything else stopping from merging this PR so that the updated RHEL support versions are listed in the docs?

**Comment by yogkumgit** [comment]
@Dansimp Gentle reminder

**Pull request merged**